main/src/main/scala/sbt/Defaults.scala

```diff
@@ -123,7 +123,7 @@ object Defaults extends BuildCommon {
       retrieveManagedSync :== false,
       configurationsToRetrieve :== None,
       scalaOrganization :== ScalaArtifacts.Organization,
-      sbtResolver := { if (sbtVersion.value endsWith "-SNAPSHOT") Classpaths.typesafeSnapshots else Classpaths.typesafeReleases },
+      sbtResolver := Classpaths.typesafeReleases,
```

Comment:
`Classpaths.typesafeSnapshots` is https://repo.typesafe.com/typesafe/ivy-snapshots/ which is https://dl.bintray.com/typesafe/ivy-snapshots/.

https://bintray.com/sbt/ivy-snapshots (which is https://repo.scala-sbt.org/scalasbt/ivy-snapshots/) isn't modeled anywhere currently. Do you want me to add it?